examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -120,20 +120,30 @@ void DeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, intptr_
     case DeviceEventType::kCommissioningComplete: {
         ESP_LOGI(TAG, "Commissioning complete");
 #if CONFIG_BT_NIMBLE_ENABLED && CONFIG_DEINIT_BLE_ON_COMMISSIONING_COMPLETE
-        int ret = nimble_port_stop();
-        if (ret == 0)
+
+        // ble_hs_is_enabled() returns 1 if ble host is enabled
+
```

Comment:
I don't think this comment is necessary. It is self explanatory.

Reply:
Okay, removed